### PR TITLE
added note about sync client and guest users

### PR DESCRIPTION
**Pull request by voroyam**
fixes https://github.com/owncloud/docs/issues/2294

added note about the guest users being able to use the sync client

### CODE REVIEW

File: modules/user_manual/pages/files/webgui/sharing.adoc [hunk 1]
```diff
@@ -86,6 +86,8 @@ permissions applied, except for the ability to share with other users.
 
 image:guest-users/content-shared-with-guest-user.png[Content shared with a guest user]
 
+NOTE: Guest users can also use the **desktop sync client** to keep their files up to date.
```
Comment (mmattel) [inline modules/user_manual/pages/files/webgui/sharing.adoc:89]:
```suggestion
NOTE: Guest users can also use the **Desktop/IOS/Andriod Sync Clients** to access and/or sync files locally.
```

**Comment by mmattel** [comment]
Backport to 10.5 and 10.6 necessary

File: modules/user_manual/pages/files/webgui/sharing.adoc (hunk 1 above)
Comment (voroyam) [reply modules/user_manual/pages/files/webgui/sharing.adoc:89]: I don't think you can access your files without synching them. By connecting your account to the server your are synching the server state to your device. I think we can leave out the and/or. 

What do you think?

Comment (mmattel) [reply modules/user_manual/pages/files/webgui/sharing.adoc:89]: good point, I have updated the suggestion. thanks!

**Pull request merged**